diff --git a/src/Appwrite/Utopia/Response/Model/Migration.php b/src/Appwrite/Utopia/Response/Model/Migration.php
index ef3c67ea6f..fd7e61ffe3 100644
--- a/src/Appwrite/Utopia/Response/Model/Migration.php
+++ b/src/Appwrite/Utopia/Response/Model/Migration.php
@@ -37,7 +37,7 @@ class Migration extends Model
             ])
             ->addRule('stage', [
                 'type' => self::TYPE_STRING,
-                'description' => 'Migration stage ( init, processing, source-check, destination-check, migrating, finished )',
+                'description' => 'Migration stage ( init, processing, source-check, destination-check, migrating, preserving_accounts, finished )',
                 'default' => '',
                 'example' => 'init',
             ])
